atomic-reactor.spec

```diff
@@ -75,9 +72,15 @@ Requires:       python3-PyYAML
 Requires:       python3-ruamel-yaml
 Requires:       python3-rpm
 Requires:       python3-six
+Requires:       koji
+Requires:       osbs >= 0.48
```

Comment:
This is not true anymore since the jsonschema for container.yaml was moved into osbs-client

Reply:
updated